lighthouserc-prod.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
 				'https://www.kiva.org/lend/filter',
 				'https://www.kiva.org/lend-by-category',
 				'https://www.kiva.org/lend-by-category/women',
-				'https://www.kiva.org/live-loan/f/sort_newest/url/1',
+				// 'https://www.kiva.org/live-loan/f/sort_newest/url/1',
```

Comment:
Unfortunately, giving the process more time didn't solve the access issues we're having trying to complete the redirect from a live loan url to an actual borrower profile page. I'm thinking now that Px is actually causing this but haven't been able to reproduce in isolation.

Reply:
What is Px?